tasty/src/dotty/tools/tasty/TastyFormat.scala

```diff
@@ -503,7 +503,6 @@ object TastyFormat {
        | DEFAULTparameterized
        | STABLE
        | EXTENSION
-       | GIVEN
```

Comment:
unrelated change ?

Reply:
Its a duplicated alternative - which is why I made https://github.com/lampepfl/dotty/issues/7643

Reply:
ok, it's best to put this sort of things in separate commits to make it clear it was intentional.

Reply:
ok I've split into 2 commits